sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/models/ExtractSummaryAction.java

```diff
@@ -0,0 +1,108 @@
+        return modelVersion;
+    }
+
+    /**
+     * Set the model version. This value indicates which model will be used for scoring, e.g. "latest", "2019-10-01".
+     * If a model-version is not specified, the API will default to the latest, non-preview version.
+     *
+     * @param modelVersion The model version.
+     *
+     * @return The {@link ExtractSummaryAction} object itself.
+     */
+    public ExtractSummaryAction setModelVersion(String modelVersion) {
+        this.modelVersion = modelVersion;
+        return this;
+    }
+
+    /**
+     * Gets the value of {@code disableServiceLogs}.
+     *
+     * @return The value of {@code disableServiceLogs}. The default value of this property is 'false'. This means,
+     * Text Analytics service logs your input text for 48 hours, solely to allow for troubleshooting issues. Setting
+     * this property to true, disables input logging and may limit our ability to investigate issues that occur.
+     */
+    public boolean isServiceLogsDisabled() {
+        return disableServiceLogs;
+    }
+
+    /**
+     * Sets the value of {@code disableServiceLogs}.
+     *
+     * @param disableServiceLogs The default value of this property is 'false'. This means, Text Analytics service logs
+     * your input text for 48 hours, solely to allow for troubleshooting issues. Setting this property to true,
+     * disables input logging and may limit our ability to investigate issues that occur.
+     *
+     * @return The {@link ExtractSummaryAction} object itself.
+     */
+    public ExtractSummaryAction setServiceLogsDisabled(boolean disableServiceLogs) {
+        this.disableServiceLogs = disableServiceLogs;
+        return this;
+    }
+
+    /**
+     * Gets the maximum extractive summarization sentences number to be returned in the response.
+     *
+     * @return The maximum extractive summarization sentences number to be returned in the response.
+     */
+    public int getMaxSentenceCount() {
+        return this.maxSentenceCount;
+    }
+
+    /**
+     * Gets the maximum extractive summarization sentences number to be returned in the response.
+     *
+     * @param maxSentenceCount The maximum extractive summarization sentences number to be returned in the response.
+     *
+     * @return The {@link ExtractSummaryAction} object itself.
+     */
+    public ExtractSummaryAction setMaxSentenceCount(int maxSentenceCount) {
+        this.maxSentenceCount = maxSentenceCount;
+        return this;
+    }
+
+    /**
+     * Gets the type of summary sentences order.
+     *
+     * @return The type of summary sentences order.
+     */
+    public SummarySentencesOrder getSentencesOrderBy() {
+        return sentencesOrderBy;
+    }
+
+    /**
+     * Sets the type of summary sentences order.The default value is {@link SummarySentencesOrder#OFFSET}.
+     *
+     * @param sentencesOrderBy The type of summary sentences order.The default value is
+     * {@link SummarySentencesOrder#OFFSET}.
```

Comment:
The default value is
     * {@link SummarySentencesOrder#OFFSET} and {what it means}.

Reply:
If user doesn't pass the value, then service will use OFFSET as the default value

Reply:
Yes, but we should probably explain what ordering by offset means.